src/db/migrate.rs

```diff
@@ -381,6 +381,25 @@ pub fn migrate(version: Option<Version>, conn: &mut Client) -> CratesfyiResult<(
             -- Nope, this is a pure database fix, no going back.
             "
         ),
+        migration!(
+            context,
+            // version
+            16,
+            // description
+            "Create new table for doc coverage",
+            // upgrade query
+            "
+            CREATE TABLE doc_coverage (
+                release_id INT NOT NULL REFERENCES releases(id),
+                total_items INT,
+                documented_items INT
+            );
+            ",
+            // downgrade query
+            "
+            -- None.
```

Comment:
This needs to drop the table, the migration downgrade fails because migration 1 removes the `releases` table, leaving the foreign key here dangling.